netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  command = "yarn install --frozen-lockfile && yarn build && cd docs && yarn install --frozen-lockfile && yarn build"
```

Comment:
Any reason for not using the `yarn build:docs` command here?

Reply:
Ignorance but also the un-frozen lockfile?

Reply:
Alright :+1: